R/estimate_dispersion.R

```diff
@@ -114,19 +144,27 @@ estimate_dispersion <- function(
     # Remove the last i observations
     trunc_matr_observed <- list_of_obs[[i]]
     max_t <- nrow(trunc_matr_observed)
+
     # Take the reporting triangle and look at one row at a time, which
     # corresponds to one horizon
     for (d in 1:n_horizons) {
-      obs_row <- trunc_matr_observed[max_t - d + 1, ]
-      nowcast_row <- nowcast_i[max_t - d + 1, ]
+      obs <- trunc_matr_observed[(max_t - d - k + 2):(max_t - d + 1), ]
```

Comment:
Motivation?

Reply:
If you don't add the look back (`k-1`) rows you are only looking at a single horizon time (row), whereas we need to compute this using `k` rows.